packages/web-client/src/helpers/space/types.ts

```diff
@@ -75,3 +75,8 @@ export interface PublicSpaceResource extends SpaceResource {
 export const isPublicSpaceResource = (resource: Resource): resource is PublicSpaceResource => {
   return resource.driveType === 'public'
 }
+
+export interface PropfindResult {
```

Comment:
Again, the term `propfind` is something API specific, in this case webdav. We want to avoid leaking API specifics as much as possible. Please use a name like `ListFilesResult`.

Apart from that, the code location of the interface is not good. This is no space type. Please export the type from `web-client/src/webdav/listFiles.ts`.

Reply:
Need to un-resolve this, sorry. Please see second part of the comment above - interface needs to be moved to the `listFiles` file.

Also, again, please don't use `propfind` in naming. There are still locations in the code left. A simple `git grep "propfind"` will show you the locations.